File: oddcheck/oddcheck.cpp
#include <iostream>

int main() 
{
    int number;
    std::cin >> number;
    if (number%2 == 1) 
    {
        std::cout << "Number is odd" << std::endl;
    }
    else 
    {
        std::cout << "Number is even" << std::endl;
    }
    return 0;
}

File: oddcheck/oddcheck.java
import java.util.Scanner;

public class oddcheck {

    public static void main(String[] args) {
        System.out.println("Please input a number");
		Scanner s = new Scanner(System.in);
		
		int number = s.nextInt();
		
		if(number % 2 == 1) {
			System.out.println(number + " is odd");
		}else {
			System.out.println(number + " is even");
		}
		
	}

}
